src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -973,6 +975,9 @@ class LicenceService(
         }
     }
   }
+  private fun hasOffenderResentensedWithActiveLicense(sentenceDatesRequest: UpdateSentenceDatesRequest, licenceEntity: EntityLicence): Boolean {
+    return (sentenceDatesRequest.conditionalReleaseDate?.isAfter(LocalDate.now()) == true || sentenceDatesRequest.actualReleaseDate?.isAfter(LocalDate.now()) == true) && licenceEntity.statusCode == ACTIVE
+  }
```

Comment:
Instead of an OR condition, check for actualReleaseDate being present. If so, use that value, otherwise use the conditionalReleaseDate. it's possible (but unlikely) the conditionalReleaseDate could be in the future, but the actualReleaseDate may be in the past. 

The actualReleaseDate is the definitive date, otherwise we rely on the conditionalReleaseDate.

I would possibly add a single check at the top of the function:
`IF (licenceEntity.statusCode !== ACTIVE`) return false`

To simplify the logic.